modules/@apostrophecms/login/index.js

```diff
@@ -253,15 +253,14 @@ module.exports = {
             const site = (req.headers.host || '').replace(/:\d+$/, '');
             const email = self.apos.launder.string(req.body.email);
             if (!email.length) {
-              throw self.apos.error('invalid');
+              throw self.apos.error('invalid', req.t('apostrophe:loginResetEmailRequired'));
+            }
+            let user;
+            try {
+              user = await self.getPasswordResetUser(req.body.email);
+            } catch (e) {
+              self.apos.util.error(e);
```

Comment:
Please just add a comment to the first of these ("error not reported to browser for security reasons") so we don't wonder later